numba/tests/test_sys_monitoring.py

```diff
@@ -285,6 +283,14 @@ def test_disable_from_callback(self):
             self.assertEqual(result, self.call_foo_result)
             callback.assert_called()
         finally:
+            # It is necessary to restart events that have been disabled. The
+            # "disabled" state of the `PY_START` event for the tool
+            # `self.tool_id` "leaks" into subsequent tests. These tests then end
+            # up failing as events that should trigger do not! It's not really
```

Comment:
```
            # up failing as events that should trigger do not! It's not really
                                                      ^^^^^^^
```
you mean `that should not trigger`?

Reply:
The comment is trying to convey that "subsequent tests fail as events in them that should be triggered are not being triggered" (this was why the `Mock.call_count` value was 1 too small, an event that should have triggered did not).

Reply:
My english parser segfaulted when reading it.

Reply:
I've tried to make this comment more clear in 25535b3. Please take a look and see what you think. Thanks!